sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+Altae-Tran et al developed a one-shot learning network
+[@tag:AltaeTran2016_one_shot] to address the reality that most practical
+chemical screening studies are unable to provide the thousands or millions of
+training compounds that are needed to train larger multitask networks.  Using
+graph convolutions to featurize chemicals, the network learns an embedding from
+compounds into a continuous feature space such that compounds with similar
+activities in a set of training tasks have similar embeddings.  The approach is
+evaluated in an extremely challenging setting where the embedding is learned
+from a subset of prediction tasks (e.g. activity assays for individual proteins)
+and only one to ten labeled examples are provided as training data on a new
+task.  On Tox21 targets, even when trained with _one_ task-specific active
+compound and _one_ inactive compound, the model is able generalize reasonably
+well because it has learned an informative embedding function from the related
+tasks. Random forests, which cannot take advantage of the related training
+tasks, trained in the same setting achieve near random performance.  Despite the
+success on Tox21, performance on MUV datasets, which contains assays designed to
+be challenging for chemical informatics algorithms, is considerably worse.  The
+authors also demonstrate the limitations of transfer learning as embeddings
+learned from the Tox21 assays have little utility for a drug adverse reaction
+dataset.
+
+In the long term, these novel, learned chemical feature representations may
+prove to be essential for accurately predicting why some compounds with similar
+structures yield similar target effects and others produce drastically different
+results.  Currently, these methods are enticing but do not necessarily
+outperform classic approaches by a large margin.  The neural fingerprints
+[@tag:Duvenaud2015_graph_conv] were narrowly beaten by regression using
+traditional circular fingerprints on a drug efficacy prediction task (but were
+superior for predicting solubility or photovoltaic efficiency).  Graph
+convolutions [@tag:Kearnes2016_graph_conv] performed comparably to a multitask
+network using standard fingerprints and slightly better than the neural
+fingerprints [@tag:Duvenaud2015_graph_conv] on the drug efficacy task but were
+slightly worse than the influence-relevance voter method on an HIV dataset.
+[@tag:Swamidass2009_irv].  `TODO: there are also problems with some papers using
+ROC primarily for benchmarking, which skews results and makes it hard to assess
+absolute performance, but I already alluded to that above and may leave further
+discussion to the Discussion section`
+
+We remain optimistic for the potential of deep learning and specifically
+representation learning in this domain and propose that rigorous benchmarking on
+broad and diverse prediction tasks will be as important as novel neural network
```

Comment:
I'm hedging because I'm not sure.  Likely both.  I feel more confident that benchmarking, code sharing, and community platforms are going to be important.